### PR TITLE
feat(Accordion): Accordion Component TypeScript Conversion

### LINKED ISSUE #2033: Convert Accordion to TypeScript
https://github.com/patternfly/patternfly-react/pull/2121

### PR DESCRIPTION
**Pull request by mturley**
Closes #2033

### CODE REVIEW

File: packages/patternfly-4/react-core/src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/Accordion/accordion.css';
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/Accordion/Accordion.tsx]: Change to `@patternfly/react-styles/css/components/Accordion/accordion`

File: packages/patternfly-4/react-core/src/components/Accordion/AccordionContent.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/Accordion/accordion.css';
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/Accordion/AccordionContent.tsx]: Change to `@patternfly/react-styles/css/components/Accordion/accordion`

File: packages/patternfly-4/react-core/src/components/Accordion/AccordionToggle.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/Accordion/accordion.css';
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/Accordion/AccordionToggle.tsx]: Change to `@patternfly/react-styles/css/components/Accordion/accordion`

**Comment by mturley** [comment]
Thanks @redallen, I'll update this today if I can (traveling right now) or first thing Monday if I can't

**Comment by mturley** [comment]
Squashed, rebased, and updated the style imports. Should be good to go now

**Comment by mturley** [comment]
Rebased and resolved a merge conflict

**Pull request merged**